File: examples-sdk/csharp/204-llm-streaming-translator/Main.cs
// 204-llm-streaming-translator — C# SDK equivalent
// Compile: vil compile --from csharp --input 204-llm-streaming-translator/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("llm-streaming-translator", 3103);
var translator = new ServiceProcess("translator");
server.Service(translator);
server.Compile();

File: examples-sdk/kotlin/204-llm-streaming-translator/main.kt
// 204-llm-streaming-translator — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 204-llm-streaming-translator/main.kt --release

fun main() {
    val server = VilServer("llm-streaming-translator", 3103)
    val translator = ServiceProcess("translator")
    server.service(translator)
    server.compile()
}
